todo_app/app_test.py

```diff
@@ -0,0 +1,81 @@
+import json
```

Comment:
It would be nice to put your tests in their own folder so they are more clearly distinguished from your actual app code.

Reply:
I was a bit confused on how to do this, would appreciate some feedback on how to go about restructuring?